iidm/iidm-serde/src/test/java/com/powsybl/iidm/serde/DcGroundSerDeTest.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2025, Coreso SA (https://www.coreso.eu/) and TSCNET Services GmbH (https://www.tscnet.eu/)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.serde;
+
+import com.powsybl.iidm.network.DcGround;
+import com.powsybl.iidm.network.DcNode;
+import com.powsybl.iidm.network.Network;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.time.ZonedDateTime;
+
+import static com.powsybl.iidm.serde.IidmSerDeConstants.CURRENT_IIDM_VERSION;
+
+/**
+ * @author Damien Jeandemange {@literal <damien.jeandemange at artelys.com>}
+ */
+class DcGroundSerDeTest extends AbstractIidmSerDeTest {
+
+    @Test
+    void testNetworkDcGround() throws IOException {
+        Network network = createBaseNetwork();
+
+        // Test for the current version
+        allFormatsRoundTripTest(network, "/dcGroundRoundTripRef.xml", CURRENT_IIDM_VERSION);
```

Comment:
Can you add in your SerDe tests a control on previous supported versions:
```java
        // backward compatibility
        allFormatsRoundTripFromVersionedXmlFromMinToCurrentVersionTest("/dcGroundRoundTripRef.xml", IidmVersion.V_1_15);
```

For this IIDM version, it won't do anything, but it will be effective for all versions > 1.15 (without having to change those tests).